src/action/gridfs/rename.rs

```diff
@@ -7,11 +7,11 @@ impl GridFsBucket {
     /// error if the `id` does not match any files in the bucket.
     ///
     /// `await` will return [`Result<()>`].
-    pub fn rename(&self, id: Bson, new_filename: impl AsRef<str>) -> Rename {
+    pub fn rename(&self, id: Bson, new_filename: impl Into<String>) -> Rename {
```

Comment:
unrelated, but it'd be nice to accept `impl Into<Bson>` for these kinds of inputs to allow passing in something like an objectID directly. I can file a ticket for that as well as that sounds good

Reply:
Yup, thank you!